ui/component/notificationContentChannelMenu/view.jsx

```diff
@@ -32,12 +31,10 @@ export default function NotificationContentChannelMenu(props: Props) {
   }
 
   return (
-    <MenuItem onSelect={handleClick}>
-      <div className="menu__link">
-        <Icon aria-hidden icon={notificationsDisabled ? ICONS.BELL : ICONS.BELL_ON} />
-        {notificationsDisabled ? __('Turn Back On') : __('Turn Off')}
-      </div>
-      <span className="menu__link-help">{claimName}</span>
```

Comment:
The channel name `claimName` in the "toggle notification" is gone.  Probably ok if it's a conscious choice or have reasons, but thought it was a nice touch to have it.

<img width="222" alt="image" src="https://user-images.githubusercontent.com/64950861/153530276-227aed6e-0960-412e-926a-05d960350f82.png"> ---> <img width="208" alt="image" src="https://user-images.githubusercontent.com/64950861/153530295-c0e353cb-34f2-4bce-9aad-92fa31d633e9.png">